handbook/engineering/engineering-management.md

```diff
@@ -49,20 +49,44 @@ Here are some books that multiple people on the team have read and recommend. If
 - [The Five Dysfunctions of a Team](https://www.amazon.com/Five-Dysfunctions-Team-Leadership-Fable/dp/0787960756) - A model of how to think about what makes a highly effective team paired with a fictional story that illustrates the model.
 - [The Leader’s Guide to Unconscious Bias](https://www.amazon.com/Leaders-Guide-Unconscious-Bias-High-Performing/dp/1982144319) ([facilitated small group discussions](../support/support-values-enablement.md#part-2-deep-dive-on-be-welcoming-and-inclusive).
 
-## Temporary Management Positions
+## Promotion requests
 
-We create temporary management positions when there is an organizational need. These may be filled by someone who is transitioning into the role (Interim Manager), experimenting with the role as they work on determining their career path (Acting Manager) or, someone who is just filling in and who is not interested in pursuing an Engineering Manager role long term (Acting Manager). This difference should be made explicit with the individual and team members before the temporary role is assigned.
+To be documented.
 
-### Acting Manager
+## Compensation increase requests
 
-An Acting Manager occupies a role temporarily and will move back to their original position after a set amount of time or other conditions. An Acting Manager may be experimenting with the role as a part of determining their career development path or could be filling in for a vacant role while we hire.
+To be documented.
 
-### Interim Manager
+## Transitioning from Individual Contributor to Engineering Manager
 
-Interim Manager positions are for teammates pursuing an Engineering Manager role long term but lack the necessary people management experience. The requirement is that, before moving into the role full time, they will make at least one successful hire. The official transition to Engineering Manager will not occur before 30 days after that new hire's start date, so that we can assess whether the hire was truly successful. If the new hire's success is indeterminate at the 30-day mark, we will continue to review until a firm decision is made. If the new hire is not successful, that does not mean that the Interim Manager cannot eventually move into the full-time role.
+For teammates pursuing an Engineering Manager role but who lack the necessary people management experience, we have defined the following transitioning guide that affords the candidate a "manager in training" opportunity to try out being an Engineering Manager for a trial period. The trial period will allow the teammate to determine if they want to commit to it long-term and verify that they would be good in the role.
```

Comment:
Verbiage edit - feel free to take/leave what you'd like @jeanduplessis:

For teammates interested in becoming Engineering Managers who may not have prior experience in people management, we have defined a transition framework that affords this candidate a "Manager in training" opportunity for a trial period. This trial period will allow the teammate to gain exposure to management principles, determine if they would excel as an Engineering Manager, and help them envision committing to this career path long-term.

Reply:
Resolving feedback in https://github.com/sourcegraph/about/pull/4303/files